test/access-assignment-spec.js

```diff
@@ -1,27 +1,9 @@
 var expect = require('expect.js');
-var simple = require('simple-mock');
-var fs = require('fs');
 var testHelper = require('../etc/test-helper.js');
-var errorFormatter = testHelper.validationErrorFormatter;
-
-// Load the contents of the sync function file into a global variable called syncFunction
-/*jslint evil: true */
-eval('var syncFunction = ' + fs.readFileSync('build/sync-functions/test-access-assignment-sync-function.js').toString());
-/*jslint evil: false */
-
-// Placeholders for stubbing built-in Sync Gateway support functions.
-// More info: http://developer.couchbase.com/mobile/develop/guides/sync-gateway/sync-function-api-guide/index.html
-var requireAccess;
-var channel;
-var access;
 
 describe('User and role access assignment:', function() {
   beforeEach(function() {
     testHelper.init('build/sync-functions/test-access-assignment-sync-function.js');
-
-    requireAccess = simple.stub();
-    channel = simple.stub();
-    access = simple.stub();
```

Comment:
In #62, the function stubs here were exposed to the outside world via the test-helper module, so there's no longer any need to stub them separately here.